### DISCUSSION TITLE
ProtonVPN instead of PIA

### DISCUSSION THREAD

**Discussion by WillScarlettOhara** [AdrienPoupa/docker-compose-nas#106] [Q&A]
Hi,
If I want to replace PIA with ProtonVPN, should I use gluetun? How to setup an auto port forwarding solution?
Thank you

**Comment by AdrienPoupa** [comment]
Yes, you can use Gluetun or any other VPN you'd like, it's documented here: https://github.com/AdrienPoupa/docker-compose-nas?tab=readme-ov-file#customization
